### ISSUE TITLE
Setting to disable in-line auto-complete suggestions.

### ISSUE THREAD

**Issue by TheCyaniteProject** [Exafunction/CodeiumVisualStudio#60]
The auto-completion is not very useful for me, and just slows down my workflow.

Being able to disable this feature would also let users whom are experiencing issues able to opt-out until the related issues are fixed.

**Comment by JanPanthera** [comment]
For my case, an option to disable automatic auto-completetion and a way to trigger it manually would be nice.

**Comment by level1cr3** [comment]
Disable automatic auto-completetion and a way to trigger it manually when i'm stuck or i need help. Pleaase provide this feature.

**Comment by jdussail** [comment]
Yes, please. I think this is truly useful and important. It was working well, but now it frequently suggests huge chunks of repeated lines, so I need to disable auto-completion.
